tsl/test/sql/continuous_aggs_ddl.sql

```diff
@@ -550,8 +551,24 @@ SELECT user_view,
  WHERE mat_table::text = table_name
    AND user_view::text LIKE 'whatever_view%';
 
+-- Checking that the access method for the materialized hypertable is
+-- set to the correct access method.
+CREATE MATERIALIZED VIEW access_view_1 USING heap2
+WITH (timescaledb.continuous, timescaledb.materialized_only=true) AS
+SELECT time_bucket('2', time), COUNT(data)
+  FROM whatever GROUP BY 1;
+
+-- Compare the materialized table for one using an access method with
+-- one not using an access method.
+SELECT user_view, mat_table, amname
+FROM cagg_info,
+     LATERAL (SELECT relam FROM pg_class WHERE relname = mat_table) AS cls,
+     LATERAL (SELECT amname FROM pg_am WHERE oid = relam) AS am
+WHERE user_view::text IN ('whatever_view_1', 'access_view_1');
```

Comment:
Could you also add a test where this view is also used with the access method

Reply:
Added.